### DISCUSSION TITLE
It's giving an error in the package "libcrystalhd-dev" it says it's not available

### DISCUSSION THREAD

**Discussion by KelvinVicctor** [ant-media/Ant-Media-Server#6724] [FAQs]
![erro](https://github.com/user-attachments/assets/87873068-2f1b-4c18-86d8-c64c7279ba90)

**Comment by muratugureminoglu** [comment]
Hi @KelvinVicctor 

If you are trying to install Ubuntu 22.04 or later and Ant Media Server version lower than 2.6, you may experience this problem. Could you please provide the output of the following command and the version of Ant Media Server you are trying to install? 
```
lsb_release -a
```

We recommend installing Ant Media Server version 2.6 and above for Ubuntu 22.04 and above.

Thanks.

**Reply by rahul78275** [reply]
Hello @muratugureminoglu 

I had the same issue with AMS Enterprise version 2.8.2 with WSL on windows.

![image](https://github.com/user-attachments/assets/a37635f4-6138-4867-b810-4079ca33c87d)

I am trying to downgrade the OS to 22.0

**Update: - On 22.04 working fine.**

**Reply by KelvinVicctor** [reply]
> Oi@KelvinVicctor
> 
> Se você estiver tentando instalar o Ubuntu 22.04 ou posterior e a versão do Ant Media Server inferior a 2.6, você pode ter esse problema. Você poderia fornecer a saída do comando a seguir e a versão do Ant Media Server que você está tentando instalar?
> 
> ```
> lsb_release -a
> ```
> 
> Recomendamos instalar o Ant Media Server versão 2.6 e superior para Ubuntu 22.04 e superior.
> 
> Obrigado.

 o problema foi resolvido, obrigado

**Reply by muratugureminoglu** [reply]
Thanks @rahul78275 Let me check on our end.

**Reply by muratugureminoglu** [reply]
Hi @rahul78275 

Before that, such installations actually pass CI tests, but I wanted to do a manual test again just in case. 

I installed AMS version 2.8.2 on Ubuntu 24.04 without any problems. 

If you want to debug the current structure, you can run the installation script with the -x parameter 

My advice is to proceed with a clean install.  

![image](https://github.com/user-attachments/assets/022f8915-e090-41f1-ba48-4275b59b21c5)

Thanks.

**Reply by rahul78275** [reply]
Thank You @muratugureminoglu for re-testing it for me.

For the experiment I cleaned my existing AMS and re-installed 24.04 on WSL. This time it worked well without any error 😲 

I don't know why it returned "libcrystalhd-dev" error last time.

**Comment by KelvinVicctor** [comment]
resolvido o problema obrigado
